<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceMeshingWidget</class>
 <widget class="QWidget" name="SurfaceMeshingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_9">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>VTK File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="sm_InputFile">
       <property name="text">
        <string>*.vtk</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="sm_InputFileBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sm_InputFileIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/icons/images/Icons.qrc">:/Delete-16x16.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Output Directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sm_OutputDir"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="sm_OutputDirBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_18">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Existing Output Files</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sm_NodesFileIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/icons/images/Icons.qrc">:/Delete-16x16.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="sm_NodesFile">
       <property name="text">
        <string>nodes.txt</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="sm_NodesRawFile">
       <property name="text">
        <string>nodes_raw.txt</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sm_NodesRawFileIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/icons/images/Icons.qrc">:/Delete-16x16.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sm_TrianglesFileIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/icons/images/Icons.qrc">:/Delete-16x16.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sm_TrianglesFile">
       <property name="text">
        <string>triangles.txt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="4">
      <spacer name="horizontalSpacer_28">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="sm_Message">
     <property name="text">
      <string>Message:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="sm_SmoothMesh">
     <property name="text">
      <string>Smooth Surface Mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_10">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Number of Iterations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sm_SmoothIterations">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="sm_LockQuadPoints">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Lock Quad Points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Write output file every </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sm_WriteOutputFileIncrement">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2" colspan="2">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>169</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QProgressBar" name="sm_progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sm_GoBtn">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
